README.md

```diff
@@ -1,7 +1,54 @@
-# BlockScalars
+# MultilineStrings
 
-[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://invenia.github.io/BlockScalars.jl/stable)
-[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://invenia.github.io/BlockScalars.jl/dev)
-[![Build Status](https://travis-ci.com/invenia/BlockScalars.jl.svg?branch=master)](https://travis-ci.com/invenia/BlockScalars.jl)
+[![Stable](https://img.shields.io/badge/docs-stable-blue.svg)](https://invenia.github.io/MultilineStrings.jl/stable)
+[![Dev](https://img.shields.io/badge/docs-dev-blue.svg)](https://invenia.github.io/MultilineStrings.jl/dev)
+[![Build Status](https://travis-ci.com/invenia/MultilineStrings.jl.svg?branch=master)](https://travis-ci.com/invenia/MultilineStrings.jl)
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
 [![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
+
+Tooling for manipulating multiline strings.
+
+## Features
+
+The package features the `@m_str` macro was inspired from [YAML's block scalars](https://yaml-multiline.info/) which provide options for manipulating multiline strings via a style and chomp indicator:
+
+- Style indicator:
+    - `f` replace newlines with spaces (folded)
+    - `l` keep newlines (literal)
+- Chomp indicator:
+    - `s` no newlines at the end (strip)
+    - `c` single newline at the end (clip)
+    - `k` keep all newlines from the end (keep)
+
+The indicators are provided after the ending quote of the string (e.g. `m"hello\nworld!"fc`).
```

Comment:
the use of the indicators is super cool.
Very under-used feature with string macros.

Reply:
Thanks. After reading the Julia documentation again I realized they aren't documented... https://docs.julialang.org/en/v1/manual/metaprogramming/#Non-Standard-String-Literals